Index: cmake-build-debug/.ninja_log
===================================================================
diff --git a/cmake-build-debug/.ninja_log b/cmake-build-debug/.ninja_log
--- a/cmake-build-debug/.ninja_log	
+++ b/cmake-build-debug/.ninja_log	
@@ -49,3 +49,6 @@
 1400	1563	1661861442910013850	UkkoShakki	1f74f35ca1149690
 1	1386	1661861465073837497	CMakeFiles/UkkoShakki.dir/src/Game.cc.o	5ec44bbc61a307d2
 1386	1560	1661861465245836190	UkkoShakki	1f74f35ca1149690
+1	790	1662406052523421364	CMakeFiles/UkkoShakki.dir/src/main.cc.o	d5ad745dc500613c
+1	913	1662406052643421109	CMakeFiles/UkkoShakki.dir/src/Game.cc.o	5ec44bbc61a307d2
+913	1073	1662406052803420769	UkkoShakki	1f74f35ca1149690
Index: cmake-build-debug/Testing/Temporary/LastTest.log
===================================================================
diff --git a/cmake-build-debug/Testing/Temporary/LastTest.log b/cmake-build-debug/Testing/Temporary/LastTest.log
--- a/cmake-build-debug/Testing/Temporary/LastTest.log	
+++ b/cmake-build-debug/Testing/Temporary/LastTest.log	
@@ -1,3 +1,3 @@
-Start testing: Sep 04 22:56 EEST
+Start testing: Sep 06 04:26 EEST
 ----------------------------------------------------------
-End testing: Sep 04 22:56 EEST
+End testing: Sep 06 04:26 EEST
